### DISCUSSION TITLE
Post execution rerun mechanism for only failed tests

### DISCUSSION THREAD

**Discussion by cj7282** [cypress-io/cypress#27044] [Questions and Help]
In our current regression testing via Cypress we are using the "retries" functionally to run failed tests one time during our test execution if they fail. 
We are wanting to add even more robustness in filtering out flaky tests by utilizing a mechanism that collects the tags of the tests that failed after the retry and reruns them at the end after the regression suite has finished running all tests. 
Once these failed tests are identified and rerun, if they pass, the report would update so we have a more true view of the failing tests that need to be modified. 

1. Does this mechanism already exist in Cypress as it is found in other frameworks that we have used in the past? (Previously used Gauge)
2. If not, are there any plans to create this in the near future?

**Comment by smeiring** [comment]
Any update or answer on this?
